### DISCUSSION TITLE
Help needed implementing a minimal working build

### DISCUSSION THREAD

**Discussion by davidkron** [jreleaser/jreleaser#869] [Q&A]
I am currently having a look at JReleaser but I am having trouble implementing a minimal working build.
What I did so far:
1) created a new project with `gradle init`
2) added the jreleaser gradle plugin to build.gradle
3) created a `jreleaser.yml` file with the minimal configuration from https://jreleaser.org/guide/latest/examples/simple-release.html

This is my build.gradle file:
```
plugins {
    id 'org.jreleaser' version '1.1.0'
}

jreleaser {
    dryrun = true
    configFile = 'jreleaser.yml'
}
```

Running `./gradlew jreleaserRelease` gets me the following error:
```
$ ./gradlew jreleaserRelease

> Configure project :
[INFO]  Reading configuration

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'gradle-init'.
> Cannot query the value of property 'licenseUrl' because it has no value available.
```

I did try setting the licenseUrl in jreleaser.yml under project, but this didn't have any effect.
What am I missing?

**Comment by aalmiray** [comment]
@davidkron I gave it a try and it looks to me that the use of `configFile` is not working as expected. As an alternative I'd suggest using the DSL directly on the Gradle build file for the time being.

**Reply by aalmiray** [reply]
Filed https://github.com/jreleaser/jreleaser/issues/870 as this is a bug (likely a regression) as I no longer see in code that the `configFile` property is used to populate the model as expected.

**Reply by davidkron** [reply]
Thank you for the confirmation. I did try the Gradle DSL directly without `configFile` and it did indeed work as expected.
